examples/hosts/app-integration/external-controller/src/view.ts

```diff
@@ -44,7 +45,7 @@ export function renderDiceRoller(diceRoller: IDiceRollerController, div: HTMLDiv
  * @param audience - Object that provides the list of current members and listeners for when the list changes
  * @param div - The div to render into
  */
-export function renderAudience(audience: ITinyliciousAudience, div: HTMLDivElement) {
+export function renderAudience(audience: ITinyliciousAudience | IFrsAudience, div: HTMLDivElement) {
```

Comment:
I'm not necessarily opposed to supporting both audience types here as a shortcut, but it's taking advantage of the fact that `TinyliciousMember` and `FrsMember` just so happen to be equal today.  If they were to diverge, I think you'd be looking at a `renderTinyliciousAudience()` + `renderFrsAudience()` (and that would be OK and expected).

Reply:
Yep! They are uniform today but they may diverge tomorrow and there is intentionally not a restriction on that, as different services are expected to have different user details. And when they do, you are absolutely correct that we'd have different renders for the different properties, i.e. if we have OdspClient in here too, then we might want to add a facepile

Reply:
This is a good reason why we should not put the burden on the developer to use both imo